src/openapi/lab.yml

```diff
@@ -154,17 +165,54 @@ model:
                 writeOnly: true
               - description: Credentials path in Vault
                 type: string
+                maxLength: 256
         example:
           hostname: ns.example.com
           zone: example.com.
           key: kv/region/rdu2-a/dns
       vault_server:
         type: string
+        maxLength: 256
         example: https://vault.example.com
       download_server:
         type: string
+        maxLength: 256
         example: https://download.example.com
 
+  ClusterStatus:
+    type: string
+    nullable: true
+    enum:
+      - Active
```

Comment:
If this will not impact negatively anything, I'd put these members in alphabetical order, so it would be easier in the future search for a specific member already added to the `enum`. Same for the `required` fields. Besides that, PR LGTM.

Reply:
It will work, regardless of the order. Currently, it is sorted how it is defined in the code. I use `git grep` or search/highlight feature in Vim to search for something in a code so I don't care if it is sorted or not.

@brhatiga @daniellutz What is your opinion? To sort, or not to sort?

Reply:
Usually I do sort, because, from my perspective, when you are editing a file, you have your first visual look on the file, and if you cannot understand exactly what is present and where, this can lead to confusion, people can add duplicated things (have seen this a lot), so, from my perspective, I do sort it, and not only YAML attributes, but CSS attributes as well on the front-end.

Reply:
While I don't always do it, I also find sorting makes it easier to spot irregularities.